examples/discovery-search-app/package.json

```diff
@@ -23,8 +23,8 @@
   },
   "dependencies": {
     "@carbon/icons": "^10.5.0",
-    "@ibm-watson/discovery-react-components": "file:../../packages/discovery-react-components",
-    "@ibm-watson/discovery-styles": "file:../../packages/discovery-styles",
+    "@ibm-watson/discovery-react-components": "workspace:*",
+    "@ibm-watson/discovery-styles": "workspace:*",
```

Comment:
yarn v2 supports a new protocol `workspace:`, which finds a package in the local workspaces, rather than pulling from remote repo. Using `*` tells yarn to find a package by the given name.

See https://yarnpkg.com/features/protocols#workspace